src/model/core/EthqlStorage.ts

```diff
@@ -105,6 +108,8 @@ class EthqlStorage {
       return web3.utils.toHex(
         web3.utils.toBN(web3.utils.sha3('0x' + this.pad(tempBase, web3))).add(web3.utils.toBN(step.query)),
       );
+    } else {
+      return step.query;
```

Comment:
Why was this added?

Reply:
Function was returning null if there was no matching storageType. Now it returns the query it was given. Changed for completeness more than anything. Doesn't change functionality.